tools/merge_schedule

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/perl
+
+# Tool to merge YAML_SCHEDULE_DEFAULT  with YAML_SCHEDULE_FLOWS
+# and YAML_SCH§EDULE.
+
+# So far all pointers should be provided as environment
+# variables
+
+use strict;
+use warnings;
+
+use File::Basename;
+use testapi qw(get_var set_var diag);
+use Env;
+
+use YAML::PP;
+use YAML::PP::Schema::Include;
+use YAML::PP::Common 'PRESERVE_ORDER';
+use scheduler;
+
+my $test_suite_data;
+my $root_project_dir = dirname(__FILE__) . '/../';
+my $include = YAML::PP::Schema::Include->new(paths => ($root_project_dir));
+my $ypp = YAML::PP->new(
+    schema => ['Core', $include, 'Merge'],
+    preserve => PRESERVE_ORDER
+);
+$include->yp($ypp);
+
+set_var('YAML_SCHEDULE', $ENV{YAML_SCHEDULE});
+set_var('YAML_SCHEDULE_FLOWS', $ENV{YAML_SCHEDULE_FLOWS});
+set_var('YAML_SCHEDULE_DEFAULT', $ENV{YAML_SCHEDULE_DEFAULT});
+
+
+my $yamlfile = get_var('YAML_SCHEDULE');
+my $schedule_file = $ypp->load_file($root_project_dir . $yamlfile);
+my %schedule_vars = scheduler::parse_vars($schedule_file);
+my @schedule_modules = scheduler::parse_schedule($schedule_file);
```

Comment:
Looks like we don't need to initialize those variables (?)
just 
```
scheduler::parse_vars($schedule_file);
scheduler::parse_schedule($schedule_file);
```
should work here

Reply:
Agreed.